### PR TITLE
Bit of cleanup for today's soft release

### LINKED ISSUE #69: Kernel spec in template.ipynb
After running a jupyterbook build on the current main version of Pythia Foundations, I got a warning that the kernel spec that is in template.ipynb was not found:

> jupyter_client.kernelspec.NoSuchKernel: No such kernel named conda-env-pythia-book-dev-py

Currently, the kernelspec is set as follows in template.ipynb:

> "kernelspec": {
   "display_name": "Python [conda env:pythia-book-dev]",
   "language": "python",
   "name": "conda-env-pythia-book-dev-py"

Our other notebooks currently use "python3" for the kernel, and doing so for template.ipynb makes for a warning-free jupyter book build. 

Yet, our instructions mention creating and activating the "pythia-book-dev" environment ...

### LINKED ISSUE #77: "30-second intro to OOP" should be moved out of datetime notebook
As discussed in #38, the section in the datetime notebook ["30-second intro to Object Oriented Programming"](https://foundations.projectpythia.org/core/datetime/datetime.html#thirty-second-introduction-to-object-oriented-programming) belongs somewhere else earlier in the Foundations sequence.

To be dealt with later...

### PR DESCRIPTION
**Pull request by dcamron**
Trimmed some file names, cleaned up some template adherence, cleared out unnecessary metadata, ran all the notebooks on the same kernel, cleared outputs and blank cells, reorganized a bit of the pandas material (@mgrover1 make sure you're okay with these changes!), and removed the OOP section in `datetime` (@brian-rose make sure this was a clean break, I didn't double check for any md references to this section). 

- [X] Closes #69

### CODE REVIEW

**Comment by brian-rose** [comment]
> removed the OOP section in `datetime` (@brian-rose make sure this was a clean break, I didn't double check for any md references to this section).

I'll take a look at the text, but the link checker should catch any broken references.

**Comment by dcamron** [comment]
Right, I just meant references within the text. I didn't see any immediately before or after, but just wanted to run it by you since you've had eyes on it.

File: core/datetime/datetime.ipynb [hunk 1]
```diff
@@ -37,14 +37,13 @@
    "source": [
```
Comment (brian-rose) [inline core/datetime/datetime.ipynb:37]: <p>Remove item 2, no more OOP</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: core/pandas/pandas.ipynb [hunk 2]
```diff
@@ -2,33 +2,58 @@
  "cells": [
```
Comment (brian-rose) [inline core/pandas/pandas.ipynb:2]: <p>different from above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

**Review by brian-rose — COMMENTED** [review]
I noted over in ReviewNB the one place I saw a stale reference to the OOP section, to be removed. Otherwise, looks great. I think the additions to the Pandas section add a lot of clarity.

**Comment by brian-rose** [comment]
@dcamron you may want to merge in the latest from main, which now includes the Xarray material from #33. Might need one or two tweaks for consistency with the other notebooks.

**Comment by dcamron** [comment]
Yep, have already rebased and am running through those as well. I will likely just delete `nc-cf` if that's alright with you so it doesn't throw off any builds or anything in the future and just open an issue about that content. We'll always have it in the history and I have python-training copies as well. Also I might rename a few more things but maybe another time when more content goes in.

**Review by mgrover1 — APPROVED** [review]
This looks solid @dcamron - thanks for making these changes

**Review by brian-rose — APPROVED** [review]

**Comment by brian-rose** [comment]
ha ha, approved after merge! 

Thanks for the cleanup effort

**Pull request merged**